package.json

```diff
@@ -14,11 +14,11 @@
   },
   "dependencies": {
     "@babel/compat-data": "^7.21.5",
-    "@bcrs-shared-components/approval-type": "1.1.2",
+    "@bcrs-shared-components/approval-type": "1.1.3",
     "@bcrs-shared-components/base-address": "2.0.3",
     "@bcrs-shared-components/breadcrumb": "2.1.15",
     "@bcrs-shared-components/business-lookup": "1.3.4",
-    "@bcrs-shared-components/certify": "2.1.15",
+    "@bcrs-shared-components/certify": "2.1.51",
```

Comment:
The new shared component changed "is authorized" to "are authorized" and fixed the input colours per accessibility standards.

Before:
![image](https://github.com/user-attachments/assets/6ca2ad30-f5d7-42d3-8d6a-0c1c6b500ae0)

After:
![image](https://github.com/user-attachments/assets/a6796413-3081-4d07-bdd1-9e71e2fd9471)